windows-server-container-tools/Debug-ContainerHost/Debug-ContainerHost.ps1

```diff
@@ -57,6 +57,9 @@ Describe "Docker is installed" {
             $filesToDump["docker version"] = "dockerversion.txt"            
         } | Should Not Throw
     }
+    It "Docker is registered in the EventLog service" {
+        (Test-Path "HKLM:\SYSTEM\CurrentControlSet\Services\EventLog\Application\docker") -or (Test-Path "HKLM:\SYSTEM\CurrentControlSet\Services\EventLog\Application\com.Docker.Service") | Should Be $true 
```

Comment:
Tested with Docker 4 Windows Beta 30.1, the `com.Docker.Service` should be `DockerService`